.editorconfig

```diff
@@ -0,0 +1,270 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space
+tab_width = 4
+
+# New line preferences
+end_of_line = crlf
+insert_final_newline = false
```

Comment:
I think I actually want this `true` - I've been trying to add trailing newlines throughout the solution as I see them missing

Reply:
done :)